port/CMakeLists.txt

```
@@ -444,23 +444,21 @@ target_link_libraries(omrport
 		j9hashtable
 		j9pool
 		${OMR_THREAD_LIB}
+		${CMAKE_DL_LIBS}
 )
 
-if(OMR_HOST_OS STREQUAL "linux")
-    target_link_libraries(omrport PRIVATE rt)
+if(OMRPORT_OMRSIG_SUPPORT)
+	target_link_libraries(omrport PRIVATE omrsig)
 endif()
 
-#TODO hack to get to compile. Need platform checks
-if(NOT OMR_HOST_OS STREQUAL "win")
-	if(NOT OMR_HOST_OS STREQUAL "zos")
-		target_link_libraries(omrport PRIVATE dl)
-	endif()
+if(OMR_OS_LINUX)
+    target_link_libraries(omrport PRIVATE rt)
```

switch to tab indentation
```suggestion
	target_link_libraries(omrport PRIVATE rt)
```